docs/modelserving/data_plane/data_plane.md

```diff
@@ -0,0 +1,60 @@
+# Data Plane
+The InferenceService Data Plane architecture consists of a static graph of components which coordinate requests for a single model. Advanced features such as Ensembling, A/B testing, and Multi-Arm-Bandits should compose InferenceServices together.
+
+## Introduction
+KServe's data plane protocol introduces an inference API that is independent of any specific ML/DL framework and model server. This allows for quick iterations and consistency across Inference Services and supports both easy-to-use and high-performance use cases.
+
+By implementing this protocol both inference clients and servers will increase their utility and 
+portability by operating seamlessly on platforms that have standardized around this API. Kserve's inference protocol is endorsed by NVIDIA
+Triton Inference Server, TensorFlow Serving, and ONNX Runtime Server.
```

Comment:
ONNX Runtime Server is no longer maintained, let’s add torchserve instead

Reply:
done